src/datatypes/gameplay/State.cpp

```diff
@@ -95,5 +141,7 @@ namespace spy::gameplay {
         }
     }
 
-
+    unsigned int State::getMoveDistance(const util::Point &p1, const util::Point &p2) {
+        return std::max(abs(p1.x - p2.x), abs(p1.y - p2.y));
```

Comment:
```suggestion
        return std::max(std::abs(p1.x - p2.x), std::abs(p1.y - p2.y));
```
wenn `std::max` dann auch `std::abs`